crates/tuono/Cargo.toml

```
@@ -37,6 +37,10 @@ spinners = "4.1.1"
 console = "0.15.10"
 
 [dev-dependencies]
+wiremock = "0.6.2"
 tempfile = "3.14.0"
 assert_cmd = "2.0.16"
 serial_test = "0.10.0"
+
+[features]
+integration_test = []
```

These changes allow `wiremock` to only be needed for testing.

```suggestion
integration_test = ["wiremock"]
```